Failure cluster [a17a6f1f...]: TestPrepareResources/should_timeout

### Failure cluster [a17a6f1fb487640e6e63](https://go.k8s.io/triage#a17a6f1fb487640e6e63)

##### Error text:
```
Failed
=== RUN   TestPrepareResources/should_timeout
    manager_test.go:609: 
        	Error Trace:	/home/prow/go/src/k8s.io/kubernetes/pkg/kubelet/cm/dra/manager_test.go:609
        	Error:      	Error "NodePrepareResources failed: rpc error: code = DeadlineExceeded desc = stream terminated by RST_STREAM with error code: CANCEL" does not contain "NodePrepareResources failed: rpc error: code = DeadlineExceeded desc = context deadline exceeded"
        	Test:       	TestPrepareResources/should_timeout
--- FAIL: TestPrepareResources/should_timeout (0.03s)

=== RUN   TestPrepareResources
--- FAIL: TestPrepareResources (0.11s)

```
#### Recent failures:
[5/22/2025, 2:46:31 PM ci-kubernetes-unit](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit/1925532299156262912)
[5/21/2025, 8:35:37 AM ci-kubernetes-unit](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit/1925077045096747008)
[5/20/2025, 4:28:16 PM ci-kubernetes-unit](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit/1924833385868431360)


/kind failing-test
/kind flake
/sig node
/wg device-management

--- a/pkg/kubelet/cm/dra/manager_test.go
+++ b/pkg/kubelet/cm/dra/manager_test.go
@@ -505,7 +505,7 @@ func TestPrepareResources(t *testing.T) {
 			claim:                genTestClaim(claimName, driverName, deviceName, podUID),
 			wantTimeout:          true,
 			expectedPrepareCalls: 1,
-			expectedErrMsg:       "NodePrepareResources failed: rpc error: code = DeadlineExceeded desc = context deadline exceeded",
+			expectedErrMsg:       "NodePrepareResources failed: rpc error: code = DeadlineExceeded",
 		},
 		{
 			description:            "should prepare resource, claim not in cache",
@@ -671,7 +671,7 @@ func TestUnprepareResources(t *testing.T) {
 			claimInfo:              genTestClaimInfo(claimUID, []string{podUID}, true),
 			wantTimeout:            true,
 			expectedUnprepareCalls: 1,
-			expectedErrMsg:         "context deadline exceeded",
+			expectedErrMsg:         "NodeUnprepareResources failed: rpc error: code = DeadlineExceeded",
 		},
 		{
 			description:            "should fail when driver returns empty response",

